### PR TITLE
removes torch dependencies in utils.voxel

### PR DESCRIPTION
**Pull request by hello-lamsey**
## Description

This PR removes dependencies on `torch_geometric`, `torch_cluster`, and `torch_scatter` in `stretch.utils.voxel`.

## Checklist

- \[ \] I have performed a self-review of my code
- \[ \] If it is a core feature, I have added thorough tests
- \[ \] I have added documentation for the changes
- \[ \] I have updated the README file if necessary
- \[ \] I have run on hardware if necessary

## Screenshots (if applicable)

Add any relevant screenshots or screen recordings to help reviewers understand the changes.

## Additional context

Add any other context or information about the pull request here.

### CODE REVIEW

File: src/stretch/utils/torch_cluster/torch_cluster_helpers.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        import torch
+        from torch_cluster import grid_cluster
+
+        pos = torch.Tensor([[0, 0], [11, 9], [2, 8], [2, 2], [8, 3]])
+        size = torch.Tensor([5, 5])
+        cluster = grid_cluster(pos, size)
+    """
+
+    # TODO: fix me!
+    return torch.ops.torch_cluster.grid(pos, size, start, end)
```
Comment (hello-lamsey) [inline src/stretch/utils/torch_cluster/torch_cluster_helpers.py:45]: This could be a pain to fix

**Comment by hello-lamsey** [comment]
need to add license declarations for 3rd parties to these files

File: src/stretch/utils/torch_cluster/torch_cluster_helpers.py (hunk 1 above)
Comment (hello-cpaxton) [reply src/stretch/utils/torch_cluster/torch_cluster_helpers.py]: This still requires torch_cluster?

**Review by hello-binit — APPROVED** [review]
LGTM

**Pull request merged**